pkgs/tools/misc/locate-dominating-file/default.nix

```
@@ -0,0 +1,56 @@
+{ bats
+, bash
+, fetchFromGitHub
+, lib
+, stdenvNoCC
+, getopt
+}:
+let
+  version = "0.0.1";
+in
+stdenvNoCC.mkDerivation {
+  pname = "locate-dominating-file";
+  inherit version;
+  src = fetchFromGitHub {
+    owner  = "roman";
+    repo   = "locate-dominating-file";
+    rev    = version;
+    sha256 = "sha256-gwh6fAw7BV7VFIkQN02QIhK47uxpYheMk64UeLyp2IY=";
```

```suggestion
    owner = "roman";
    repo = "locate-dominating-file";
    rev = version;
    sha256 = "sha256-gwh6fAw7BV7VFIkQN02QIhK47uxpYheMk64UeLyp2IY=";
```

No need for pretty tabulation.

---

:facepalm: I copy pasted that from the previous file without running `nixpkgs-fmt`, done that now